lib/utils/gestures.html

```diff
@@ -466,7 +466,7 @@
      * @private
      * @param {!HTMLElement} node Node on which to add the event.
      * @param {string} evType Event type to add.
-     * @param {function(Event?)} handler Event handler function.
+     * @param {function(!Event)} handler Event handler function.
```

Comment:
@aomarks instead of changing `addListener` to take `!Event=`, i just changed `_add` to not be optional. this internally ends up doing `addEventListener` and i dont think theres an event which would fire without an event...